Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -145,12 +148,19 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         _enableCreateDeleteDraftButton: function () {
```

Comment:
it should be renamed to match what it does now for instance `_setDraftButtonsState` (and the doc description should be updated)